sympy/__init__.py

```diff
@@ -35,10 +35,8 @@ def enable_warnings():
 
 
 import sys
-if ((sys.version_info[0] == 2 and sys.version_info[1] < 7) or
-    (sys.version_info[0] == 3 and sys.version_info[1] < 4)):
-    raise ImportError("Python version 2.7 or 3.4 or above "
-                      "is required for SymPy.")
+if sys.version_info < (3, 5):
+    raise ImportError("Python 3.5 or above is required for SymPy.")
```

Comment:
Are you making a backward-incompatible change?

Reply:
Yes, the idea is to drop support for Python 2.7: https://github.com/sympy/sympy/issues/17943.

Reply:
1.5.1 will be the last version to support Python 2. See https://github.com/sympy/sympy/wiki/Python-version-support-policy